lib/cipherstash/client.rb

```diff
@@ -153,6 +153,7 @@ def create_collection(name, schema)
             meta: {
               "$indexId" => SecureRandom.uuid,
               "$indexName" => idx_name,
+              # TODO: we don't need two keys for filter indexes
```

Comment:
Also, for the avoidance of confusion, we don't want to call it `$prfKey`.  `$filterKey` would be a nice name for it, IMO.